diagnostic_feedback/quiz_result.py

```diff
@@ -0,0 +1,94 @@
+import collections
+
+
+class StudentResultPresenter(object):
+    html_body = ''
+
+    def __init__(self, **data):
+        self.html_body = data.get('html_body', '')
+
+    def generate_result(self):
+        return {_property: getattr(self, _property) for _property in dir(self) if not _property.startswith('__')
+                and not callable(getattr(self, _property))}
```

Comment:
This is giving a PEP8 error: `quiz_result.py:12:17: W503 line break before binary operator`